deployment/docker/update-psi-j-testing-service

```diff
@@ -84,6 +87,11 @@ fi
 
 if [ "$ANSWER" == "y" ]; then
     pip install "$PACKAGE"
+    PACKAGE_LOC=`pip show $PACKAGE_NAME | grep "Location: " | sed 's/Location: //'`
+    LINK_NAME="$PACKAGE_LOC/psij/web/instance/customization.js"
+    if [ ! -a "$LINK_NAME" ]; then
```

Comment:
`[ -a filename ] ` : test if a  filename exists
`[ ! -a filename ] ` : test if `not` and test if `filename`

In other words: the second `-a` acts as binary operator, i.e., as logical `and`.  I think this is not what you want here?

```
$ touch foo
$ rm -f bar
$ [ -a foo ] && echo 'yes foo'
yes foo
$ [ -a bar ] && echo 'yes bar'
$ [ ! -a foo ] && echo 'no foo'
no foo
$ [ ! -a bar ] && echo 'no bar'
no bar 
```

You probably want `-e` instead.

Reply:
Nice catch. Thanks. Fixed.